sass/layout/_sidebar-menu.scss

```diff
@@ -107,3 +107,10 @@
         }
     }
 }
+.app__body {
+    .modal-tabs {
+        .nav-tabs > li {
+            background-color: var(--center-channel-bg);
+        }
+    }
+}
```

Comment:
@jgilliam17, ditto to the one above ... I honestly can't find this anywhere in actual use anymore. @devinbinnie, @hmhealey, how about you guys?

Reply:
In the context of `modal-tabs` I can't find reference to it either. Would be worth making sure it's not used by `QuickSwitchModal` but otherwise we can remove.

Reply:
I investigated the QuickSwitchModal. `modal-tabs` and `nav-tabs` don't seem to be used together here or anywhere else. Additionally, the tabs in the `QuickSwitchModal` are manually disabled (`components/quick_switch_modal/index.tsx`, ln. 14).

Just wanted to make sure I didn't miss anything 😛